python_modules/libraries/dagster-dg/dagster_dg/cli/scaffold.py

```diff
@@ -189,6 +196,168 @@ def scaffold_workspace_command(
     scaffold_workspace(name, workspace_config)
 
 
+# ########################
+# ##### GITHUB ACTIONS
+# ########################
+
+
+def _search_for_git_root(path: Path) -> Optional[Path]:
```

Comment:
Somewhere, a customer still using git submodules is lurking

Reply:
but I don't know what to suggest; possibly an option to specify the git root path?

Reply:
Added an option to specify git root, but it's assumed by default